gulpfile.js

```diff
@@ -7,6 +7,34 @@ const sassGlob = require('gulp-sass-glob');
 const cssnano = require('gulp-cssnano');
 const sourcemaps = require('gulp-sourcemaps');
 
+const iconfont = require('gulp-iconfont');
+
+const pficonRunTimestamp = Math.round(Date.now() / 1000);
+const iconfontCss = require('gulp-iconfont-css');
+
+const pficonFontName = 'pficon';
+
+gulp.task('build-pficonfont', () => {
+  gulp
+    .src(['./src/icons/PfIcons/*.svg'])
+    .pipe(
+      iconfontCss({
+        fontName: pficonFontName,
+        targetPath: 'pficon.css',
+        fontPath: './',
+        cssClass: 'pficon'
+      })
+    )
+    .pipe(
+      iconfont({
+        fontName: pficonFontName,
+        formats: ['ttf', 'eot', 'woff', 'woff2', 'svg'],
+        timestamp: pficonRunTimestamp
+      })
+    )
+    .pipe(gulp.dest('./src/patternfly/assets/pficon/'));
```

Comment:
The gulp scripts were originally intended only to create the dist output for the library.  Will this output be needed by the workspace when developing?  If so - we might need to have a script to make sure the dev environment is populated and we copy to dist as a separate step.

Reply:
Ok, this is fine for now - generally we'd like to avoid checking in anything that is compiled.  That just complicates the dev experience so we can just ignore that issue for now and deal with that later.